src/system/file_path.cc

```diff
@@ -40,7 +40,7 @@ FilePath::FilePath(string file_path, const  FileType ft)
     if (ft == input_file) {
     	if ( FilePath::is_absolute_path(abs_file_path_) ) {
```

Comment:
Could we implement FilePath::is_absolute_path using boost::filesystem::path.is_absolute()? Just to be as consistent as possible. Then some calls of this method may be replaced by direct call of the boost is_absolute. E.g. consider

	FilePath::is_absolute_path(output_dir)
vs.
	output_path.is_absolute()